.github/workflows/_buildpacks-release-publish-docker.yml

```diff
@@ -1,65 +0,0 @@
-name: _buildpacks-release-publish-docker
-
-on:
-  workflow_call:
-    inputs:
-      buildpack_id:
-        required: true
-        type: string
-        description: The id of the buildpack to package
-      buildpack_artifact_prefix:
-        required: true
-        type: string
-        description: The prefix name to use for any generated buildpack artifacts
-      buildpack_version:
-        required: true
-        type: string
-        description: The version of the buildpack to package
-      docker_repository:
-        required: true
-        type: string
-        description: The docker repository to publish the buildpack to
-      dry_run:
-        required: false
-        type: boolean
-        default: false
-        description: Flag used for testing purposes that prevents the actual publish to docker
-    secrets:
-      docker_hub_user:
-        required: true
-        description: The username to login to Docker Hub with
-      docker_hub_token:
-        required: true
-        description: The token to login to Docker Hub with
-
-jobs:
-  release-publish-docker:
-    name: Publish → Docker
-    runs-on: ubuntu-latest
-    steps:
-      - name: Restore buildpack release artifacts
-        id: artifacts
-        uses: heroku/languages-github-actions/.github/actions/restore-buildpack-release@main
```

Comment:
Similarly, with this removal, `.github/actions/restore-buildpack-release` is now unused and can be deleted too :-)